### PR TITLE
feat/QC-5371-final-small-changes

### PR DESCRIPTION
**Pull request by thomaskosciuch**
please dont make fun of my lacklustre tests. more are coming.

### CODE REVIEW

File: cdk_build/DigitFtpStack.py [hunk 1]
```diff
@@ -20,11 +20,15 @@ class Digit2NbinStack(Stack):
     """
 
     cron_schedule: aws_events.Schedule = aws_events.Schedule.cron(
-        minute='10', hour='*', month='*', week_day='*', year='*')
+        minute='/5', hour='19', month='*', week_day='MON-FRI', year='*') #3pm in UTC
 
-    def cron(self, aws_events, cycle) -> aws_events:
+    cron_schedule_2: aws_events.Schedule = aws_events.Schedule.cron(
+        minute='0', hour='12-18', month='*', week_day='MON-FRI', year='*') #9-5 in UTC
```
Comment (thomaskosciuch) [inline cdk_build/DigitFtpStack.py]: should be 9-2 in UTC

File: cdk_build/DigitFtpStack.py [hunk 2]
```diff
@@ -20,11 +20,15 @@ class Digit2NbinStack(Stack):
     """
 
     cron_schedule: aws_events.Schedule = aws_events.Schedule.cron(
-        minute='10', hour='*', month='*', week_day='*', year='*')
+        minute='/5', hour='19', month='*', week_day='MON-FRI', year='*') #3pm in UTC
```
Comment (thomaskosciuch) [inline cdk_build/DigitFtpStack.py]: every 5 minutes from 3:00 to 3:55

File: cdk_build/DigitFtpStack.py [hunk 3]
```diff
@@ -20,11 +20,15 @@ class Digit2NbinStack(Stack):
     """
 
     cron_schedule: aws_events.Schedule = aws_events.Schedule.cron(
-        minute='10', hour='*', month='*', week_day='*', year='*')
+        minute='/5', hour='20', month='*', week_day='MON-FRI', year='*') #3pm in UTC
```
Comment (QW-Matt) [inline cdk_build/DigitFtpStack.py:23]: /5 for minute?

**Review by QW-Matt — APPROVED** [review]

**Comment by thomaskosciuch** [comment]
approved by matt (he said so in slack): 

image

**Pull request merged**